network/client.go

```diff
@@ -0,0 +1,112 @@
+package network
```

Comment:
Since we emulate network, we must emulate clients also, so it have to have access to the local network stack... And more basically, it's related to network operations so it should be there IMO :)